config/initializers/devise.rb

```diff
@@ -0,0 +1,312 @@
+# frozen_string_literal: true
+
+require "omniauth-facebook"
+require "omniauth-twitter"
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '664fc90f13ede6525aaeb9ab5e2aa1a61f4b55a87d3bab6add3'
+
+  # ==> Controller configuration
+  # Configure the parent class to the devise controllers.
+  # config.parent_controller = 'DeviseController'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = "please-change-me-at-config-initializers-devise@example.com"
```

Comment:
Please change me??

Reply:
Yep, I don't know anything about this I'm afraid. This is code from James.

Reply:
I guess this should be a valid SMV reply address? I'm not sure it's used.

Reply:
It is definitely used in the reset password emails, just checked

Reply:
To test it, send yourself an email from https://swapmyvotedev.herokuapp.com/users/password/new

Reply:
Done.